docker-compose.yml

```diff
@@ -20,6 +20,8 @@ notarysigner:
    - notarymysql
   command: -config=fixtures/signer-config.json
 notarymysql:
+  volumes:
+    - notarymysql:/var/lib/mysql
```

Comment:
careful with that, because that may change in future (because of "named" volumes). If you want to create a bind-mounted volume, using `./notarymysql/:/var/lib/mysql/` may be safer. Compose will translate `./..` to relative to the `docker-compose.yml`